metrics/ci_integrations/test/source/jenkins/adapter/jenkins_source_client_adapter_test.dart

```diff
@@ -371,6 +421,53 @@ void main() {
       },
     );
 
+    test(
+      ".fetchBuildsAfter() maps builds that are building to in-progress builds",
```

Comment:
```suggestion
      ".fetchBuildsAfter() maps running builds to in-progress builds",
```
Please apply to all test cases in this PR.